docs/tutorials/networks_seq2seq_nmt.ipynb

```diff
@@ -108,19 +108,6 @@
         "# **Setup**"
```

Comment:
<p>move line 9 to line 6 and remove trailing empty lines</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/908/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>